src/primitives/pureheader.cpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2019 The Bitcoin Core developers
+// Copyright (c) 2021 The Dogecoin developers
+// Distributed under the MIT/X11 software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <primitives/pureheader.h>
+
+#include <crypto/scrypt.h>
+#include <hash.h>
+#include <util/strencodings.h>
+
+void CPureBlockHeader::SetBaseVersion(int32_t nBaseVersion)
```

Comment:
I'm assuming this is here instead of in the header file because later on, with AuxPoW, there will be other things added here than just setting the version bytes?

Reply:
Yup, it becomes https://github.com/namecoin/namecoin-core/blob/master/src/primitives/pureheader.cpp#L18